<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LyceumSearch</class>
 <widget class="QMainWindow" name="LyceumSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LyceumSearch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>125</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>–ü–æ–∏—Å–∫</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbKeywords">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>78</y>
      <width>110</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>–ö–ª—é—á–µ–≤—ã–µ
—Å–ª–æ–≤–∞</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextSearchField">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>100</y>
      <width>525</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblSection">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>7</y>
      <width>700</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>–í—Å–µ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetResults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>775</width>
      <height>400</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>–ù–∞–∑–≤–∞–Ω–∏–µ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>–¢–∏–ø</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>–°—Å—ã–ª–∫–∞</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSearchProblems">
    <property name="title">
     <string>–ò—Å–∫–∞—Ç—å –∑–∞–¥–∞—á–∏</string>
    </property>
    <widget class="QMenu" name="menuFirstYear">
     <property name="title">
      <string>–ü–µ—Ä–≤—ã–π –≥–æ–¥</string>
     </property>
     <addaction name="actionProblems_FY"/>
     <addaction name="actionStudentsBook_FY"/>
     <addaction name="separator"/>
     <addaction name="actionLessons_FY"/>
     <addaction name="actionIndependentWorks_FY"/>
     <addaction name="actionTests_FY"/>
     <addaction name="separator"/>
     <addaction name="actionAll_FY"/>
    </widget>
    <widget class="QMenu" name="menuSecondYear">
     <property name="title">
      <string>–í—Ç–æ—Ä–æ–π –≥–æ–¥</string>
     </property>
     <addaction name="actionProblems_SY"/>
     <addaction name="actionStudentsBook_SY"/>
     <addaction name="actionLessons_SY"/>
     <addaction name="separator"/>
     <addaction name="actionAll_SY"/>
    </widget>
    <widget class="QMenu" name="menuAll">
     <property name="title">
      <string>–í—Å–µ</string>
     </property>
     <addaction name="actionProblems_All"/>
     <addaction name="actionStudentsBook_All"/>
     <addaction name="separator"/>
     <addaction name="actionLessons_All"/>
     <addaction name="actionIndependentWorks_All"/>
     <addaction name="actionTests_All"/>
     <addaction name="separator"/>
     <addaction name="actionAll_All"/>
    </widget>
    <addaction name="menuAll"/>
    <addaction name="separator"/>
    <addaction name="menuFirstYear"/>
    <addaction name="menuSecondYear"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>üåê</string>
    </property>
    <addaction name="actionRussian"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>–ü–æ–º–æ—â—å</string>
    </property>
    <addaction name="actionOpenTutorial"/>
   </widget>
   <widget class="QMenu" name="menuOtherActions">
    <property name="title">
     <string>–î—Ä—É–≥–∏–µ –¥–µ–π—Å—Ç–≤–∏—è</string>
    </property>
    <addaction name="actionOpenFullDB"/>
    <addaction name="actionOpenDBUpdater"/>
    <addaction name="actionResetAll"/>
   </widget>
   <addaction name="menuSearchProblems"/>
   <addaction name="menuOtherActions"/>
   <addaction name="menuHelp"/>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_Fike">
   <property name="text">
    <string>–û—Ç–∫—Ä—ã—Ç—å –ø—Ä–æ–≥—Ä–∞–º–º—É</string>
   </property>
  </action>
  <action name="actionLoad_tests">
   <property name="text">
    <string>–ó–∞–≥—Ä—É–∑–∏—Ç—å —Ç–µ—Å—Ç(—ã)</string>
   </property>
  </action>
  <action name="actionIndependentWorks_FY">
   <property name="text">
    <string>–°–∞–º–æ—Å—Ç–æ—è—Ç–µ–ª—å–Ω—ã–µ</string>
   </property>
  </action>
  <action name="actionTests_FY">
   <property name="text">
    <string>–ö–æ–Ω—Ç—Ä–æ–ª—å–Ω—ã–µ</string>
   </property>
  </action>
  <action name="actionLessons_FY">
   <property name="text">
    <string>–£—Ä–æ–∫–∏</string>
   </property>
  </action>
  <action name="actionProblems_FY">
   <property name="text">
    <string>–ó–∞–¥–∞—á–∏</string>
   </property>
  </action>
  <action name="actionStudentsBook_FY">
   <property name="text">
    <string>–£—á–µ–±–Ω–∏–∫</string>
   </property>
  </action>
  <action name="actionLessons_SY">
   <property name="text">
    <string>–£—Ä–æ–∫–∏</string>
   </property>
  </action>
  <action name="actionIndependentWorks_SY">
   <property name="text">
    <string>–°–∞–º–æ—Å—Ç–æ—è—Ç–µ–ª—å–Ω—ã–µ</string>
   </property>
  </action>
  <action name="actionTests_SY">
   <property name="text">
    <string>–ö–æ–Ω—Ç—Ä–æ–ª—å–Ω—ã–µ</string>
   </property>
  </action>
  <action name="actionProblems_SY">
   <property name="text">
    <string>–ó–∞–¥–∞—á–∏</string>
   </property>
  </action>
  <action name="actionStudentsBook_SY">
   <property name="text">
    <string>–£—á–µ–±–Ω–∏–∫</string>
   </property>
  </action>
  <action name="actionLessons_All">
   <property name="text">
    <string>–£—Ä–æ–∫–∏</string>
   </property>
  </action>
  <action name="actionIndependentWorks_All">
   <property name="text">
    <string>–°–∞–º–æ—Å—Ç–æ—è—Ç–µ–ª—å–Ω—ã–µ</string>
   </property>
  </action>
  <action name="actionTests_All">
   <property name="text">
    <string>–ö–æ–Ω—Ç—Ä–æ–ª—å–Ω—ã–µ</string>
   </property>
  </action>
  <action name="actionStudentsBook_All">
   <property name="text">
    <string>–£—á–µ–±–Ω–∏–∫</string>
   </property>
  </action>
  <action name="actionProblems_All">
   <property name="text">
    <string>–ó–∞–¥–∞—á–∏</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>–†—É—Å—Å–∫–∏–π</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionMeme">
   <property name="text">
    <string>Meme</string>
   </property>
  </action>
  <action name="actionOpenTutorial">
   <property name="text">
    <string>–û—Ç–∫—Ä—ã—Ç—å –æ–±—É—á–µ–Ω–∏–µ</string>
   </property>
  </action>
  <action name="actionResetAll">
   <property name="text">
    <string>–°–±—Ä–æ—Å–∏—Ç—å –≤—Å–µ</string>
   </property>
  </action>
  <action name="actionOpenDBUpdater">
   <property name="text">
    <string>–ò–∑–º–µ–Ω–∏—Ç—å –ë–î</string>
   </property>
  </action>
  <action name="actionAll_All">
   <property name="text">
    <string>–í—Å–µ</string>
   </property>
  </action>
  <action name="actionAll_FY">
   <property name="text">
    <string>–í—Å–µ</string>
   </property>
  </action>
  <action name="actionAll_SY">
   <property name="text">
    <string>–í—Å–µ</string>
   </property>
  </action>
  <action name="actionOpenFullDB">
   <property name="text">
    <string>–ü–æ–ª–Ω–∞—è –ë–î</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
